string func_name
string pose_name
---
bool success
string[] locations